### PR TITLE
Add Clarity Global CSS 2 Client Extension - Font Changes

### PR DESCRIPTION
**Pull request by davicms**

### CODE REVIEW

File: client-extensions/liferay-clarity-global-css-2/client-extension.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+assemble:
+    - include: static/*.css
+liferay-sample-global-css:
```
Comment (smbmatheus-lr) [inline client-extensions/liferay-clarity-global-css-2/client-extension.yaml]: Please fix the Client Extension name.

File: client-extensions/liferay-clarity-global-css-2/client-extension.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    name: Liferay Sample Global CSS 2
```
Comment (smbmatheus-lr) [inline client-extensions/liferay-clarity-global-css-2/client-extension.yaml:4]: Fix CX name

**Review by smbmatheus-lr — CHANGES_REQUESTED** [review]
Hi @davicms , please fix the naming in the .yaml file. 
Thanks.

**Comment by dnebing** [comment]
@davicms the name in the client-extension.yaml file still doesn't match the folder name, still has 'sample' instead of 'clarity'.

**Pull request closed without merging**